index.html

```diff
@@ -1,11 +1,190 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
-</head>
-<body>
-  
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Homework 1</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Raleway:wght@700&family=Roboto:ital,wght@0,400;0,500;0,900;1,700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./css/style.css" />
+  </head>
+  <body>
+    <header class="header">
+      <div class="container">
+        <div class="header__content">
+          <a href="#" class="logo">
+            <span class="logo--web">Web</span>
+            <span class="logo--studio">Studio</span>
+          </a>
+          <nav class="header__nav">
+            <ul class="nav__list">
+              <li class="nav__item">
+                <a class="nav__link__studio" href="#">Студія</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">Портфоліо</a>
+              </li>
+              <li class="nav__item">
+                <a class="nav__link" href="#">Контакти</a>
+              </li>
+            </ul>
+          </nav>
+          <div class="contacts">
+            <a href="#" class="contacts__email">info@devstudio.com</a>
+            <a href="#" class="contacts__mobile">+38 096 111 11 11</a>
+          </div>
+        </div>
+      </div>
+    </header>
+    <main>
+      <section class="hero">
+        <div class="hero_container">
+          <h1 class="hero_h1">ЕФЕКТИВНІ РІШЕННЯ ДЛЯ ВАШОГО БІЗНЕСУ</h1>
+          <button name="button" class="hero_button">Замовити послугу</button>
+        </div>
+      </section>
+      <section class="middle_container">
+        <ul class="middle_container_ul">
+          <li class="middle_container_li">
+            <h3 class="middle_container_h3">УВАГА ДО ДЕТАЛЕЙ</h3>
+            <p class="middle_container_p">
+              Ідейні міркування, і навіть початок повсякденної роботи з
+              формування позиції.
+            </p>
+          </li>
+          <li class="middle_container_li">
+            <h3 class="middle_container_h3">ПУНКТУАЛЬНІСТЬ</h3>
```

Comment:
this class name can be 'features__feature-title'
```suggestion
            <h3 class="feature__title">ПУНКТУАЛЬНІСТЬ</h3>
```